core/templates/dev/head/pages/exploration-editor-page/statistics-tab/issues/answer-submit-action.directive.html

```diff
@@ -0,0 +1,11 @@
+<div ng-if="$ctrl.currentStateName == $ctrl.destStateName">
```

Comment:
Try to reduce the duplication here. I think the first two lines of each branch can be moved outside the ng-if.

Reply:
Done.